src/services/facebook.js

```diff
@@ -38,6 +38,46 @@ class Facebook {
 
     this.media = new Media(this);
     this.subscription = new Subscription(this);
+    this.cancellable = [];
+  }
+
+  /**
+   * @description plugin lifecycle, initialization
+   * @returns {Promise<void>}
+   */
+  async init() {
+    this.logger.debug('facebook service/plugin initialization.');
+
+    if (this.config.subscribeOnStart) {
+      this.logger.debug('facebook service/plugin will subscribe in 60 seconds.');
+
+      await Promise.delay(60000);
```

Comment:
why 60s

Reply:
no idea, it was taken from here 
https://github.com/makeomatic/mservice-social/blob/d0e494b5d69a846eda48dd02ee5eded882b528ad/src/social.js#L90

Reply:
I suggest it gives the service enought time to complete initialization.